<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartnodeList</class>
 <widget class="QWidget" name="SmartnodeList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer0">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_filter_2">
           <property name="text">
            <string>Filter List:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="filterLineEditDIP3">
           <property name="toolTip">
            <string>Filter smartnode list</string>
           </property>
           <property name="placeholderText">
            <string>Filter by any property (address or protx hash)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxMySmartnodesOnly">
           <property name="toolTip">
            <string>Show only smartnodes this wallet has keys for.</string>
           </property>
           <property name="text">
            <string>My smartnodes only</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_count_2">
           <property name="text">
            <string>Node Count:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="countLabelDIP3">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTableWidget" name="tableWidgetSmartnodesDIP3">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Service</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Status</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>PoSe Score</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Registered</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Last Paid</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Next Payment</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Payout Address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Operator Reward</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Collateral Address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Collateral Amount</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Owner Address</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Voting Address</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
